lib/feedback_form.dart

```diff
@@ -129,10 +160,10 @@ class _FeedbackDisplayState extends State<FeedbackDisplay> {
           var f = feedbackList[index];
           bool upvoted = f.upvotes.contains(email);
           bool downvoted = f.downvotes.contains(email);
-          // TODO: add option to delete one's own feedback
           return ListTile(
             title: Text(f.email),
             subtitle: Text(f.message),
+            onLongPress: uid == f.uid ? () => handleFeedbackLongPress(f) : null,
```

Comment:
I don't think that long press is intuitively understandable as a delete option. Maybe it's better to add some x-mark or dropdown with a delete field?

Reply:
It is not intuitive, but this is not the final design. All the designs will (or at least should) be redone later. This is just for adding the functionality.